src/plugins/trace/dll/DataModel/QuicEvents.cs

```diff
@@ -32,15 +32,16 @@ public enum QuicApiType
         ConnectionStart,
         ConnectionSetConfiguration,
         ConnectionSendResumptionTicket,
-        ConnectionCompleteResumptionTicketValidation,
```

Comment:
Does the order matter?

Reply:
The order in here must match `QUIC_TRACE_API_TYPE`